tests/Microsoft.Bot.Builder.TestBot.NetCore21.Tests/Dialogs/GetBookingDetailsDialogTests.cs

```diff
@@ -41,7 +66,9 @@ public async Task DialogFlowUseCases(TestDataObject testData)
 
             if (testClient.DialogTurnResult.Result != null)
             {
-                var bookingResults = (BookingDetails)testClient.DialogTurnResult.Result;
```

Comment:
Will this work? Or devs will need to change the code as we did in this example?

Reply:
@gabog   Reverted this casting change.  (it was required in an earlier iteration, but now is not)